record.py

```diff
@@ -143,13 +154,12 @@ def draw(self):
 
         # Joystick
         x = np.asarray(self.plotData)
+        self.axes.clear()
```

Comment:
nice! I always thought my approach for a dynamic plot was a bit sketchy

Reply:
Yeah, I didn't put much thought into it when I re-wrote the code... mostly just copied it over. Reading through it this time, I wasn't quite sure what `hold()` was even doing... until I took it out and the graph started drawing all over itself. The documentation wasn't entirely obvious either, but I figured out that `clear()` worked, for this purpose anyway.